tags_update/tests/test_verify.py

```diff
@@ -0,0 +1,41 @@
+from git_autograder import GitAutograderTestLoader, assert_output
+from git_autograder.status import GitAutograderStatus
+
+from ..verify import (
+    verify,
+    MISSING_JANUARY_TAG,
+    WRONG_APRIL_TAG_COMMIT,
+    OLD_FIRST_UPDATE_TAG,
+    SUCCESS_MESSAGE,
+    MISSING_COMMIT_MESSAGE,
+)
+
+REPOSITORY_NAME = "tags-update"
+
+loader = GitAutograderTestLoader(__file__, REPOSITORY_NAME, verify)
+
+
+def test_base():
+    with loader.load("specs/base.yml", "start") as output:
+        print(output)
+        assert_output(output, GitAutograderStatus.SUCCESSFUL, [SUCCESS_MESSAGE])
+
+
+def test_missing_tags():
+    with loader.load("specs/missing_tags.yml", "start") as output:
+        assert_output(output, GitAutograderStatus.UNSUCCESSFUL, [MISSING_JANUARY_TAG])
+
+
+def test_wrong_april_tag():
+    with loader.load("specs/wrong_april_tag.yml", "start") as output:
+        assert_output(output, GitAutograderStatus.UNSUCCESSFUL, [WRONG_APRIL_TAG_COMMIT])
```

Comment:
I think we would also want to ensure that the same check for the January tag commit should exist too, what do you think?